Gemfile.lock

```diff
@@ -54,7 +54,7 @@ GEM
     jquery-rails (3.1.2)
       railties (>= 3.0, < 5.0)
       thor (>= 0.14, < 2.0)
-    json (1.8.1)
+    json (1.8.3)
```

Comment:
"Oh, yeah, there is a known compatability issue with rails 2.x and json 1.8.1, heres a link to the article: xx.xx"